### PR TITLE
Provide easy SSH access to VM

### PR DESCRIPTION
**Pull request by mads-hartmann**
## Description

DRAFT

## Related Issue(s)

Fixes https://github.com/gitpod-io/ops/issues/643

## How to test

DRAFT

## Release Notes

```release-note
NONE
```

## Documentation

N/A

/werft with-vm

### CODE REVIEW

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **jankeromnes** after the PR has been reviewed.
You can assign the PR to them by writing `/assign @jankeromnes` in a comment when ready.

Associated issue: *#12*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/gitpod-io/gitpod/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: dev/preview/ssh-proxy-command.sh [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+port="$1"
+namespace="$2"
+
+kubectl \
+    --context=harvester \
+    --kubeconfig=/home/gitpod/.kube/config \
+    -n "${namespace}" port-forward service/proxy "${port}:22" > /dev/null 2>&1 &
+
+# Wait for the port to be read
+sleep 5
```
Comment (mads-hartmann) [inline dev/preview/ssh-proxy-command.sh:12]: Here we'd want to actually wait for SSH to be ready in a nice way, maybe use nc

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **princerachit** after the PR has been reviewed.
You can assign the PR to them by writing `/assign @princerachit` in a comment when ready.

Associated issue: *#643*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/gitpod-io/gitpod/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by mads-hartmann** [comment]
This work was completed in https://github.com/gitpod-io/gitpod/pull/7756 instead

**Pull request closed without merging**